ui/ui-components/pages/model/[modelId]/overview.tsx

```diff
@@ -36,6 +37,8 @@ const Page: NextPage<Props> = ({
   schedules,
   destinations,
 }) => {
+  if (!model) return <Alert variant="warning">Model not found</Alert>;
```

Comment:
Note - if you went to a model overview page for a model that doesn't exist, you would have previously gotten a 'not iterable error'

Reply:
I just noticed this change in main and one thing to note is that it's bad practice to return before all the hooks are set. React has a lot of problems with it.